core/src/util/yamlHelper.cpp

```diff
@@ -64,30 +64,44 @@ glm::vec4 getColorAsVec4(const Node& node) {
 }
 
 std::string parseSequence(const Node& node) {
+    if (node.IsSequence()) { return ""; }
+
     std::stringstream sstream;
     for (const auto& val : node) {
-        try {
-            sstream << val.as<double>() << ",";
-        } catch (const BadConversion& e) {
-            try {
-                sstream << val.as<std::string>() << ",";
-            } catch (const BadConversion& e) {
-                LOGE("Float or Unit expected for styleParam sequence value");
-            }
+        if (!val.IsScalar()) {
+            // LOG
+            return "";
+        }
+
+        double value;
+        if (getDouble(val, value)) {
```

Comment:
I don't understand this, we're converting a string to a double so that we can convert it back to a string? Shouldn't we just concatenate the `Scalar()` values?